staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -73,6 +73,10 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 
 	chains, err := req.TLS.PeerCertificates[0].Verify(optsCopy)
 	if err != nil {
+		switch err.(type) {
+		case x509.UnknownAuthorityError:
```

Comment:
At this level, the authenticator doesn't know there is a "next authenticator". The union chain lets the next authenticator have a try, even when this returns an error.